pkg/config/config.go

```diff
@@ -168,6 +182,7 @@ func (cli *CLI) NewFlagSet(name string) *flag.FlagSet {
 	fs.StringVar(&cli.AndroidCertFingerprint, "android-cert-fingerprint", "", "android cert fingerprint for deep linking")
 	cli.StringSliceFlag(fs, &cli.Labelers, "labelers", "", "did of labelers that this instance should subscribe to")
 	fs.StringVar(&cli.AtprotoDID, "atproto-did", "", "atproto did to respond to on /.well-known/atproto-did (default did:web:PUBLIC_HOST)")
+	fs.StringVar(&cli.ContentFiltersPath, "content-filters", "", "path to YAML file with content filtering rules")
```

Comment:
Sooo the configuration we've encouraged users to do so far has just been with environment variables, I think we'll want to stick with that for now. Yes, stuffing all of this configuration into a JSON string is ugly, but I think the remedy for that will probably be creating a proper admin configuration interface and saving it in the new stateful database I introduced here https://github.com/streamplace/streamplace/pull/456/files#diff-f30f431d4de9ce917ded96a3f035d886824f24b188769b783ef3ead0b88a9cf1

So let's just do JSON in an env var for now